examples/asr/notebooks/6_VAD_using_NeMo.ipynb

```diff
@@ -0,0 +1,991 @@
+    "\n",
+    "The notebook will follow the steps below:\n",
+    "\n",
+    " - Dataset preparation: Preparing Google Speech Commands dataset\n",
+    "\n",
+    " - Audio preprocessing (feature extraction): signal normalization, windowing, (log) spectrogram (or mel scale spectrogram, or MFCC)\n",
+    "\n",
+    " - Data augmentation using SpecAugment \"[SpecAugment: A Simple Data Augmentation Method for Automatic Speech Recognition](https://arxiv.org/abs/1904.08779)\" to increase number of data samples.\n",
+    " \n",
+    " - Develop a small Neural classification model which can be trained efficiently.\n",
+    " \n",
+    " - Model training on the Google Speech Commands dataset and Freesound dataset in NeMo.\n",
+    " \n",
+    " - Evaluation of error cases of the model by audibly hearing the samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Preparation\n",
+    "\n",
+    "## Download the Freesound dataset\n",
+    "We will be using the freesound dataset (background categories) as our non-speech/background data. \n",
+    "We provide scripts and you can customize a lot by using it. Note that downloading this dataset may takes hours. \n",
+    "\n",
+    "1. Download provided downloading and resampling scripts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/download_resample_freesound.sh\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_download.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_private_apikey.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_resample.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. We will need some requirements including freesound, requests, requests_oauthlib, joblib, librosa and sox. If they are not installed, please run "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -r freesound_requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Create an API key for freesound.org at https://freesound.org/help/developers/ and paste the cliend_id and api_key to freesound_private_apikey\n",
```

Comment:
Ordering is not coming properly in the notebook - always resets to 1,2... 

If not possible to fix, dont set numbering and just use bullet points or arrows.